lms/static/js/learner_dashboard/Static2UCallouts.jsx

```diff
@@ -3,168 +3,160 @@ import React from 'react';
 
 class Static2UCallouts extends React.Component {
 
+  constructor(props) {
+    super(props);
+    this.state = {
+      show2uLobs: false,
+    };
+  }
+
   componentDidMount() {
-    if (!this.props.isEnterpriseUser){
-      window['optimizely'] = window['optimizely'] || [];
-      window['optimizely'].push({
-        type: "page",
-        pageName: "merchandise_2u_lobs"
-      });
-    }
+    const optimizely = window["optimizely"] || [];
+    optimizely.push({
+      "type": "user",
+      "attributes": {
+        "isEnterpriseUser": this.props.isEnterpriseUser.toString(),
+      }
+    });
+    optimizely.push({
+      type: "page",
+      pageName: "van_1097_merchandise_2u_lobs_on_dashboard"
+    });
+    const experimentId = '22164741776';
+    this.timer = setTimeout(function() {
+      const selectedVariant = optimizely.get("state").getVariationMap()[experimentId];
+      if (selectedVariant.name === 'dashboard_with_2u_lobs'){
```

Comment:
if the user doesn't fall in variation then `selectedVariant` will be undefined and this code will crash.
```suggestion
      if (selectedVariant?.name === 'dashboard_with_2u_lobs'){
```